src/sqlancer/databend/DatabendErrors.java

```diff
@@ -42,6 +44,9 @@ public static List<String> getExpressionErrors() {
         if (DatabendBugs.bug9806) {
             errors.add("segment pruning failure");
         }
+        if (DatabendBugs.bug15568) {
+            errors.add("Decimal overflow at line : 723 while evaluating function `to_decimal");
```

Comment:
That's fine for this PR. Perhaps it would make the code more future proof to use regular expressions rather than line numbers.